File: modl 4/python/PRAK403-2310817210027-MuhammadErzaRaihan.py
def main():
    E, Z = map(int, input().split())
    Max = abs(E - Z)

    if E < Z:
        for i in range(Max + 1):
            j = 1
            print(f"{E} {Z} - ", end="")

            E = E + j
            Z = Z - j
    else:
        for i in range(Max + 1):
            j = 1
            print(f"{E} {Z} - ", end="")

            E = E - j
            Z = Z + j

if __name__ == "__main__":
    main()

File: modl 4/C/PRAK403-2310817210027-MuhammadErzaRaihan.c
#include <stdio.h>

int main()
{
    int E, Z;
    int Max;

    scanf("%d %d", &E, &Z);
    Max = (E-Z);    
    if (Max <0)
    {
        Max = Max*-1;
    for (int i = 0; i <= Max; i++)
    {
        int j = 1;
        printf("%d %d - ", E, Z);

        E = E+j;
        Z = Z-j;

    }
    }
    else{
    for (int i = 0; i <= Max; i++)
        {
            int j = 1;

            printf("%d %d - ", E, Z);

            E = E-j;
            Z = Z+j;
        }
    }
    return 0;
}